README.md

```diff
@@ -46,6 +46,33 @@ sudo podman run \
     quay.io/centos-bootc/fedora-bootc:eln
 ```
 
+### Using local containers
+
+To use containers from local container's storage rather than a registry, we need to ensure two things:
+- the container exists in local storage
+- mount the local container storage
+
+Since the container is run in `rootful` only root container storage paths are allowed.
+
+```bash
+sudo podman run \
+    --rm \
+    -it \
+    --privileged \
+    --pull=newer \
+    --security-opt label=type:unconfined_t \
+    -v $(pwd)/config.json:/config.json \
+    -v $(pwd)/output:/output \
+    -v /var/lib/containers/storage:/var/lib/containers/storage \
```

Comment:
Might make more sense to use DEST /usr/lib/containers/storage:ro, since I don't think we want BIB to modify the host storage, or do you plan on modifying the host storage?

/usr/lib/containers/storage should already be setup as an additional store.

Reply:
Hmm, but if we do a read-only mount, we get into issues that podman/skopeo cannot lock the store. Any thoughts about this?

Reply:
So yeah we could mount another store other than `/var/lib/containers/storage`, that's no hassle. Testing locally I was using another store entirely. So maybe we can update the docs accordingly.

The `ro` is a bit more complicated, we run into loads of locking issues. We aren't even able to generate the manifest. Do you have any suggestions for getting around that? fyi @achilleas-k

Reply:
Readonly means that the images are readonly in the addiitonal store in /usr/lib/containers/storage, while BIB would have its own /var/lib/containers/storage and able to pull images or run containers there.  There is little to protect the container from the user mucking around with the external containers-storage, so it is more of tell the user do not do that.

Reply:
However, when we use the podman/skopeo tools to inspect the containers in the container store, the tools need to put locks **inside the store**. When the store is read/only, podman/skopeo (called by BIB) cannot create the locks, and they will just fail.